config/pj-on-kind.sh

```diff
@@ -19,9 +19,9 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-export CONFIG_PATH="$(readlink -f $(dirname "${BASH_SOURCE[0]}")/../prow/config.yaml)"
-export JOB_CONFIG_PATH="$(readlink -f $(dirname "${BASH_SOURCE[0]}")/jobs)"
+export CONFIG_PATH="$(realpath $(dirname "${BASH_SOURCE[0]}")/../prow/config.yaml)"
+export JOB_CONFIG_PATH="$(realpath $(dirname "${BASH_SOURCE[0]}")/jobs)"
```

Comment:
`realpath` is not natively installed on mac FWIW.

instead this is more portable:
```suggestion
export JOB_CONFIG_PATH="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd -P)/jobs"
```

Reply:
Thanks. I'm surprised there isn't a portable command to canonicalize paths.

Reply:
Yeah, `cd $path && pwd -P` is pretty much the only method doable on POSIX, unless you write your own binary against the C stdlib.

Reply:
`pwd -P` (`-P` == "physical" will follow symlinks for the current working directory and is in all POSIX versions IIRC. `-L` == logical will not)